tests/trainers/test_segmentation.py

```diff
@@ -146,3 +147,26 @@ def test_missing_attributes(
         model = SemanticSegmentationTask(**model_kwargs)
         trainer = Trainer(fast_dev_run=True, log_every_n_steps=1, max_epochs=1)
         trainer.validate(model=model, datamodule=datamodule)
+
+    @pytest.mark.parametrize(
+        "name,classname",
+        [("deepglobelandcover_batch_size", DeepGlobeLandCoverDataModule)],
+    )
```

Comment:
Don't need to parametrize this if there's only one, but I think your other PRs will do the same thing so you'll just keep appending to this list, right?

Reply:
yes, I will append new configs for the other PRs